src/features/community/components/CommunityHUD.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useContext } from "react";
+
+import { Balance } from "components/Balance";
+import { Context } from "../lib/CommunityProvider";
+import { useActor } from "@xstate/react";
+
+export const CommunityHud: React.FC = () => {
+  const { communityService } = useContext(Context);
+  const [state] = useActor(communityService);
+
+  return (
+    <div aria-label="Hud">
+      <Balance balance={state.context.balance} />
```

Comment:
I think this will be very confusing for players. I think we should have a different balance display that includes a metamask logo or something like that. Something that clearly indicates that we're showing a balance that is different for the in game balance.

Reply:
Agreed. We can tidy up a few things pre-launch